### PR TITLE
Adding documentaion

### PR DESCRIPTION
**Pull request by LinlinCui-NOAA**
### Description
This PR added rst files for publishing documentation on `readthedocs`.

### Linked Issues
- Closes https://github.com/NOAA-EMC/graphcast/issues/27

### Input data
- [x] No changes are expected to input data.
- [ ] Changes are expected to input data:
  - [ ] New input data.
  - [ ] Updated input data.

### Needed libraries
None

### CODE REVIEW

File: NCEP/docs/source/outputs.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+######################
+Product
+######################
+
+The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles, which both applies on 13 levels and 37-levels. 
+The near real-time forecast outputs along with inputs are available on `AWS <https://noaa-nws-graphcastgfs-pds.s3.amazonaws.com/index.html>`_. For each cycle, the dataset contains input files to feed into GraphCast found in the directory ./graphcastgfs.yyyymmdd/hh/input 
+and 10-day forecast results for the current cycle found iin the directory ./graphcastgfs.yyyymmdd/hh/forecasts_13_levels and 
+./graphcastgfs.yyyymmdd/hh/forecasts_37_levels, respectively.
```
Comment (junwang-noaa) [inline NCEP/docs/source/outputs.rst:8]: @LinlinCui-NOAA Can you list the 13 and 37 pressure levels in this file? Also the output fields? Thanks

Comment (LinlinCui-NOAA) [reply NCEP/docs/source/outputs.rst:8]: @junwang-noaa Ok.

Comment (LinlinCui-NOAA) [reply NCEP/docs/source/outputs.rst:8]: @SadeghTabas-NOAA I noticed that there are only 31 levels here:
            variables_to_extract['.pgrb2.0p25.f000'][':SPFH|VVEL|VGRD|UGRD|HGT|TMP:']['levels'] = [':(1|2|3|5|7|10|20|30|50|70|100|150|200|250|300|350|400|450|500|550|600|650|700|750|800|850|900|925|950|975|1000) mb:']

**Review by SadeghTabas-NOAA — APPROVED** [review]

File: NCEP/docs/source/outputs.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+######################
+Product
+######################
+
+The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles, which applies to both 13 levels and 37-levels. The 13 pressure levels
```
Comment (junwang-noaa) [inline NCEP/docs/source/outputs.rst:5]: Can we adjust format:

The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles. The horizontal resolution is on 0.25 degree lat-lon grid. The vertical resolutions are on both 13 levels or 37 pressure levels. 

- The 13 pressure levels include:
50, 100, 150, 200, 250, 300, 400, 500, 600, 700, 850, 925, and 1000 hPa. 

- The 37 pressure levels include:
1, 2, 3, 5, 7, 10, 20, 30, 50, 70, 100, 125, 150, 175, 200, 225, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 825, 850, 875, 900, 925, 950, 975, and 1000 hPa. 

The model output fields are:

- 3D fields on pressure levels:  
 * temperature
 * U and V component of wind
 * geopotential height
 * specific humidity
 * vertical velocity
- 2D surface fields: 
   * 10-m U and V components of wind
   * 2-m temperature
   * mean sea-level pressure
   * 6 hourly total precipitation

The near real-time forecast outputs along with inputs are available on `AWS <https://noaa-nws-graphcastgfs-pds.s3.amazonaws.com/index.html>`_. 

For each cycle, the dataset contains input files to feed into GraphCast found in the directory graphcastgfs.yyyymmdd/hh/input 
and 10-day forecast results for the current cycle found in the following directories:
graphcastgfs.yyyymmdd/hh/forecasts_13_levels 
graphcastgfs.yyyymmdd/hh/forecasts_37_levels, respectively

Comment (SadeghTabas-NOAA) [reply NCEP/docs/source/outputs.rst:5]: @junwang-noaa I was wondering if this is all the comments from your side? Linlin already merged this, so I asked her to create a very small PR and apply this new comment. Please let us know if this is all?

Comment (junwang-noaa) [reply NCEP/docs/source/outputs.rst:5]: Yes. We may need to double check when we see the doc on readthedocs

**Pull request merged**